lib/bundler/resolver.rb

```diff
@@ -344,7 +344,7 @@ def verify_gemfile_dependencies_are_found!(requirements)
           versions = @source_requirements[name][name].map(&:version)
           message  = "Could not find gem '#{requirement}' in #{requirement.source}.\n"
           if versions.any?
-            message << "Source contains '#{name}' at: #{versions.join(", ")}"
+            message << "Source contains '#{name}' at: #{versions.join(", ")} #{requirement.__platform}"
```

Comment:
Gotcha. In that case, any context on how to specify gems in the test suite that have a non-ruby platform? I'm looking at the `Spec::Builders` module and the gems being built using `build_repo1`. The gems that specify ulterior platforms like so:

```
        build_gem "only_java" do |s|
          s.platform = "java"
          s.write "lib/platform_specific.rb", "PLATFORM_SPECIFIC = '1.0.0 JAVA'"
        end
```

seem to still have ruby (rather than java in this case) as the platform when I'm playing around with the following spec in `spec/install/gemfile/git_spec.rb`:

```
    it "complains if pinned specs don't exist in the git repo" do
      build_git "only_java"

      install_gemfile <<-G
        gem "only_java", "1.1", :git => "#{lib_path("only_java-1.0")}"
      G

      expect(out).to include("Source contains 'foo' at: 1.0")
    end
```